.circleci/config.yml

```diff
@@ -10,12 +10,39 @@ executors:
   mac:
     environment:
       EMSDK_NOTTY: "1"
-      # Without this, any `brew installl` command will result in self-update of
+      # Without this, any `brew install` command will result in self-update of
       # brew itself which takes more than 4 minutes.
       HOMEBREW_NO_AUTO_UPDATE: "1"
     macos:
-      xcode: "12.5.1"
+      xcode: "13.4.1"
```

Comment:
Actually does this dermine the base image to use.. or just the xcode version that is installed in the image?  If it changes the base image I think we should revert this line.

Reply:
I'm not sure. The reason I changed it though is because there is no 12.5.1 for M1, and I figured it would be good for them to match.

Reply:
It looks like it does map to the OS version: https://circleci.com/docs/using-macos/#supported-xcode-versions-intel.. so I think we should revert this line.

Reply:
ok. so you're ok with the version not matching between the different hardware?

Reply:
Sure.  I think the important thing is that we are testing on older versions of macOS where possible